### ISSUE TITLE
[content] Add Events section

### ISSUE THREAD

**Issue by Nevraeka** [webcomponents/beta#505]
1. Add a events section that shows upcoming events for web components
2. Add a featured event section for larger events like Polymer Summit & wcremoteconf.com
3. Use what we can from [Issue #206](https://github.com/webcomponents/webcomponents.github.io/issues/206) & [PR #226](https://github.com/webcomponents/webcomponents.github.io/pull/226)

**Comment by Nevraeka** [comment]
cc @obetomuniz

**Comment by abdonrd** [comment]
@Nevraeka we can add the next [Polymer Day Madrid](https://www.polymerday.com) too!

**Comment by Nevraeka** [comment]
@tjsavage - will this be addressed by UX designs? How can we go about getting this in as a feature?

**Comment by samuelli** [comment]
Seems like there's events & meetups. Is that correct?

**Comment by samuelli** [comment]
Is this just static content we can include?

**Comment by Nevraeka** [comment]
@samuelli yes - events & meetups. No & Yes...it is static content & dynamic content. 

To explain, my proposal is to create two sections basically:
* A featured section which is static content manually updated
* integration with meetup.com (and/or other event system api) to dynamically populate  a list of events directly related to web components.

How I see it...the dynamic part displays 'event cards' - maybe w/ some sorting capabilities & search. They display the event title, the event date, the event summary, the event meetup name, people involved, & a link to the event.

The challenges I see are maintaining the manual piece of this & also meetup.com sometimes has zero results for events based on web components meetups. The average returned number of events I saw while testing was 2 events, which looks very sad. I think it is a good idea to try and aggregate event data from another source to offset this. I am unsure what that would be though. Eventbrite? Lanyard? Thoughts? cc / @tjsavage
